### PR TITLE
DOC-570: add mcetableapplycellstyle cmd

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-570

Description of Changes:
* Add `mcetableapplycellstyle` cmd to table plugin cmds list
* Add 5.4 release note for `mcetableapplycellstyle` cmd

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/commands/table-cmds.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 | Command                 | Description                                     |
 | ----------------------- | ----------------------------------------------- |
+| mcetableapplycellstyle | Applies the specified styles to the selected cells. The following styles can be changed with this command: `background-color`, `border-color`, `border-style`, and `border-width`. |
```
Comment (lnewson) [inline _includes/commands/table-cmds.md:7]: We should use camel case for consistency here, eg: `mceTableApplyCellStyle`

File: release-notes/release-notes54.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ The {{site.productname}} 5.4 release includes the following improvements for the
 
 - Adds: commands, APIs, and icons for; cut, copy, and paste columns.
 - Adds toolbar button icons for the cut, copy, and paste rows.
+- Adds a new `mcetableapplycellstyle`command for applying selected styles to table cells.
```
Comment (lnewson) [inline release-notes/release-notes54.md:32]: Camel case here as well please.

File: _includes/commands/table-cmds.md [hunk 3]
```diff
@@ -29,6 +30,7 @@
 **Examples**
 
 ```js
+tinymce.activeEditor.execCommand('mcetableapplycellstyle', false, { backgroundColor: 'red', borderColor: 'blue' });
```
Comment (ltrouton) [inline _includes/commands/table-cmds.md:33]: This command will need to be in camel case here as well.

I am wondering if we need some mention on how a style can be removed with an empty string and whether that should be another example like: `tinyMCE.activeEditor.execCommand('mceTableApplyCellStyle', false, { backgroundColor: '' })` or a sentence in the description further up the page.

Comment (metricjs) [reply _includes/commands/table-cmds.md:33]: I think an example would be good here, yes, to show off what it can do

Comment (metricjs) [reply _includes/commands/table-cmds.md:33]: Maybe where we say "applies" in the various sections, we could say "toggles" or "applies and removes"? Idk

File: _includes/commands/table-cmds.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/commands/table-cmds.md]: Fixed by: https://github.com/tinymce/tinymce-docs/pull/1570/commits/82e7d51f189765c65c7b9e80f3360d50a1dee960

File: release-notes/release-notes54.md (hunk 2 above)
Comment (tylerkelly13) [reply release-notes/release-notes54.md]: Fixed by: https://github.com/tinymce/tinymce-docs/pull/1570/commits/82e7d51f189765c65c7b9e80f3360d50a1dee960

File: _includes/commands/table-cmds.md (hunk 3 above)
Comment (tylerkelly13) [reply _includes/commands/table-cmds.md]: Recommended changes made in https://github.com/tinymce/tinymce-docs/pull/1570/commits/82e7d51f189765c65c7b9e80f3360d50a1dee960

File: _includes/commands/table-cmds.md [hunk 4]
```diff
@@ -4,6 +4,7 @@
 
 | Command                 | Description                                     |
 | ----------------------- | ----------------------------------------------- |
+| mceTableApplyCellStyle | Applies the specified styles to the selected cells. The following styles can be changed with this command: `background-color`, `border-color`, `border-style`, and `border-width`. Providing an empty value for a style will remove the style, such as `{ background-color: '' }`. {{site.requires_5_4v}} |
```
Comment (ltrouton) [inline _includes/commands/table-cmds.md:7]: Looks good. Where you have, `{ background-color: '' }`, I think it should be `{ 'background-color': '' }` to be valid JS.

File: _includes/commands/table-cmds.md (hunk 3 above)
Comment (ltrouton) [reply _includes/commands/table-cmds.md]: Looks good. Thanks Tyler.

File: _includes/commands/table-cmds.md (hunk 4 above)
Comment (tylerkelly13) [reply _includes/commands/table-cmds.md]: fixed in f80bc61fabd89b13c07894d33a0047e5e0680f45

**Pull request merged**